pkg/operator/ceph/csi/spec.go

```diff
@@ -486,13 +486,23 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 	}
 
 	if EnableRBD {
-		err = createCSIDriverInfo(ctx, clientset, RBDDriverName)
+		rbdfsGroupPolicy, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_RBD_FSGROUPPOLICY", "")
```

Comment:
Nit: Go syntax guidelines suggest this should be `rbdFsGroupPolicy` or `rbdFSGroupPolicy`